### PR TITLE
isaac817's Morgan Hill file addition

### LINKED ISSUE #10: Create a new file for isaac817 Morgan Hill
- [ ] Create a branch
- [ ] Add commits
- [ ] Open a PR
- [ ] Collaborate
- [ ] Seek approval
- [ ] Merge PR

### PR DESCRIPTION
**Pull request by isaac817**
closes #10

### CODE REVIEW

**Comment by wwang-intuit** [comment]
👍

File: isaac817-morganhill.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Morgan Hill, CA
+
+### Places to Eat
+- Hamami Sushi
+- In n out
```
Comment (wwang-intuit) [inline isaac817-morganhill.md]: I love In n out

File: isaac817-morganhill.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Morgan Hill, CA
+
+### Places to Eat
```
Comment (isaac817) [inline isaac817-morganhill.md]: I too love it.

**Review by wwang-intuit — APPROVED** [review]
Perfect

**Pull request merged**